command/login.go

```diff
@@ -50,7 +51,12 @@ Login Options:
 
   -oidc-callback-addr
     The address to use for the local OIDC callback server. This should be given
-    in the form of <IP>:<PORT> and defaults to "localhost:4649".
+    in the form of <IP>:<PORT> and defaults to "localhost:4649". It is only
+    required if using OIDC auth method type. 
```

Comment:
```suggestion
    in the form of <IP>:<PORT> and defaults to "localhost:4649".
```

I think the flag name make this clear already?